resources/views/dropdown.blade.php

```diff
@@ -1,7 +1,8 @@
 @props([
-    'dropdownProperty'   => 'dropdownOpen',
+    'dropdownProperty'    => 'dropdownOpen',
     'buttonClassExpanded' => 'text-theme-primary-500',
     'buttonClass'         => 'text-theme-secondary-400 hover:text-theme-primary-500',
+    'fullScreen'          => false,
```

Comment:
I prefer adding props on top of the file. I think it's helpful to understand the component without zapping in the code to find how to use it.